### PR TITLE
Add docs for EDP 1.4.0

### PR DESCRIPTION
**Pull request by slega**
## Description
Add docs for the 1.4.0 release of the portal:
- Update deployment diagram
- Add the `db` option for the storage config option
- Add FAQ about SQL support for the portal's CMS assets to the installation page

## Motivation and Context
1.4.0 release of the portal add support for SQL as a storage for CMS assets.

## Preview Link
https://deploy-preview-2637--tyk-docs.netlify.app/docs/nightly/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/

## Types of changes

- [ ] Fixing typo (please merge to production)
- [x] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [x] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I have added a preview link.

### CODE REVIEW

**Review by zalbiraw — APPROVED** [review]
LGTM

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md [hunk 1]
```diff
@@ -69,4 +69,9 @@ Additionally, the API Analytics dashboard will be compromised.
 
 However, API traffic will remain unaffected, meaning that your APIs will continue to be operational, and analytics will continue to be recorded.
 
-In terms of admin functionality, the only limitation will be the inability to approve or reject access requests or revoke or rotate access credentials.
\ No newline at end of file
+In terms of admin functionality, the only limitation will be the inability to approve or reject access requests or revoke or rotate access credentials.
+
+
+### Does the portal support SQL database for storing the portal's CMS assets?
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md]:
```suggestion
### Does the portal support SQL databases for storing the portal's CMS assets?
```

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration.md [hunk 2]
```diff
@@ -184,47 +184,48 @@ PORTAL_S3_PRESIGN_URLS=true
 **Type:** `string` <br/>
 **Options:**
 - `fs` to use file system storage type;
+- `db` to use the portal's main database. In the `db` is selected as a storage type, the portal application will create appropriate structure in the database that is used to store the portal's metadata during [the bootstrap process]({{< ref "/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/bootstrapping-portal.md" >}});
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration.md]:
```suggestion
- `db` to use the portal's main database. If the `db` is selected as a storage type, the portal application will create appropriate structure in the database that 
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Hi Sergei. The PR looks good to me. Made some minor suggestions and committed them.

When both peers have completed their reviews let me know, then will merge and release.

**Review by dcs3spp — APPROVED** [review]

**Review by carlosvillanua — APPROVED** [review]
Looks good to me! Thank you.

**Pull request merged**